src/Products/SiteErrorLog/tests/testSiteErrorLog.py

```diff
@@ -178,3 +178,44 @@ def testCleanup(self):
         # Need to make sure that the __error_log__ hook gets cleaned up
         self.app._delObject('error_log')
         self.assertEqual(getattr(self.app, '__error_log__', None), None)
+
+import Testing.testbrowser
+import Testing.ZopeTestCase
+import Zope2.App
+import codecs
+
+from ..SiteErrorLog import manage_addErrorLog
+
+class SiteErrorLogUITests(Testing.ZopeTestCase.FunctionalTestCase):
+    
+    def setUp(self):
+        super(SiteErrorLogUITests, self).setUp()
+        
+        Zope2.App.zcml.load_site(force=True)
+
+        uf = self.app.acl_users
+        uf.userFolderAddUser('manager', 'manager_pass', ['Manager'], [])
+        
+        # Why is this neccessary, shouldn't the test get a new app every time?
+        if not hasattr(self.app, 'error_log'):
+            manage_addErrorLog(self.app)
+        self.error_log = self.app.error_log
+        
+        self.browser = Testing.testbrowser.Browser()
+        # Change back once https://github.com/zopefoundation/Zope/commit/4a6338876f15fb01f49c5b07f54e0bda39bab009 is released
+        # self.browser.login('manager', 'manager_pass')
+        self.browser.addHeader(
+             'Authorization',
+             'basic {}'.format(codecs.encode(
+                 b'manager:manager_pass', 'base64').decode()))
+        
+        self.browser.open('http://localhost/error_log/manage_main')
+    
+    def testSubmitRetainsIgnoredExceptionsUnchanged(self):
+        # Checks the fix for https://github.com/zopefoundation/Products.SiteErrorLog/issues/13
+        ignoredExceptions = self.browser.getControl(label='Ignored exception types')
+        self.assertEquals(ignoredExceptions.value, 'Unauthorized\nNotFound\nRedirect') # default value
+        self.browser.getControl('Save Changes').click()
```

Comment:
It would be nice if the test actually changes something as this would show that the stored value is read back.

Is there a text rendered after successfully submitting the form? There should be an assertion that it is there to make sure we do not look at the page showing an error message instead of saving.